src/js/keyboardInput.js

```diff
@@ -128,7 +135,39 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             "onCreate.registerChangeListener": {
                 funcName: "gpii.firstDiscovery.keyboardInput.registerChangeListener",
                 args: ["{that}", "{that}.container"]
+            },
+            // begin TOOLTIP HANDLER CONFIGURATION
+            //
+            // We want to control the tooltip opening ourselves so we
+            // remove the existing mouseover and focusin handlers and
+            // add our own instead.  We leave the mouseleave and
+            // focusout handlers alone as the jQuery Tooltip widget
+            // rebinds these each time the tooltip is opened.
+            "onCreate.removeMouseover": {
+                "this": "{that}.container",
+                method: "off",
+                args: ["mouseover"],
+                priority: 4
+            },
+            "onCreate.removeFocusin": {
+                "this": "{that}.container",
+                method: "off",
+                args: ["focusin"],
+                priority: 3
+            },
+            "onCreate.mouseoveHandler": {
+                "this": "{that}.container",
+                method: "mouseover",
+                args: ["{that}.openTooltipIfNotFocused"],
+                priority: 2
+            },
+            "onCreate.focusinHandler": {
+                "this": "{that}.container",
+                method: "focusin",
+                args: ["{that}.tooltip.close"],
+                priority: 1
```

Comment:
Instead of doing this with 4, 3, 2, 1, could you just use "first" and "last"? I assume that the sets can fire in any order.